src/test/java/io/confluent/idesidecar/restapi/kafkarest/api/ClusterV3Suite.java

```diff
@@ -2,14 +2,14 @@
 
 import static org.hamcrest.Matchers.equalTo;
 
-import io.confluent.idesidecar.restapi.util.AbstractSidecarIT;
+import io.confluent.idesidecar.restapi.integration.ITSuite;
 import io.confluent.idesidecar.restapi.util.ConfluentLocalKafkaWithRestProxyContainer;
 import org.junit.jupiter.api.Test;
 
-abstract class ClusterV3ApiImplIT extends AbstractSidecarIT {
+public interface ClusterV3Suite extends ITSuite {
```

Comment:
This is an example of one of these `*IT` classes that was extended by a `*LocalIT` class and a `*DirectIT` class. The `*IT` class had all of the test methods, so the class was changed as follows:
* The `*IT` abstract base class was made an interface that extends `ITSuite` rather than `AbstractSidecarIT` -- these are runnable sets of test classes now, and instead are merely interfaces that define test methods.
* The `@Test` methods were turned into default methods that depend only on the methods (some abstract) inherited from `ITSuite`

Reply:
The `LocalITs` and `ConfluentPlatformITs` classes now contain the test classes which get the test methods from one or more of these "suite" interfaces.